easybuild/easyconfigs/b/Boost/Boost-1.69.0-GCCcore-11.3.0-no_mpi.eb

```diff
@@ -0,0 +1,31 @@
+name = 'Boost'
+version = '1.69.0'
+versionsuffix = '-no_mpi'
```

Comment:
@Micket, If there was no extension then this version would be picked up in preference to the `GCC `and `intel-compilers` versions.  I'm happy to change to a different suffix if you can think of something more appropriate.

Reply:
I don't understand what difference the suffix is supposed to do here. It will conflict (as it should) with the other versions regardless, the version string is already a unique identifier.